packages/harmony/src/components/layout/layout.module.css

```diff
@@ -0,0 +1,34 @@
+.row {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+  
+  .col {
+    display: flex;
+    flex-direction: column;
+  }
+  
+  .gap1 {
+    gap: var(--unit-1);
+  }
+  .gap2 {
+    gap: var(--unit-2);
+  }
+  .gap3 {
+    gap: var(--unit-3);
+  }
+  .gap4 {
+    gap: var(--unit-4);
+  }
+  .gap5 {
+    gap: var(--unit-5);
+  }
+  .gap6 {
+    gap: var(--unit-6);
+  }
+  
+  .gapAuto {
+    justify-content: space-between;
+  }
+  
```

Comment:
can we try using the Flex component to cover these cases instead of classes?

Reply:
Oh yes ty. Great callout. I started these changes before you merged yours haha so I'll just remove these and adjust accordingly